sdk/typescript/src/providers/json-rpc-provider.ts

```diff
@@ -312,8 +316,11 @@ export class JsonRpcProvider extends Provider {
   }
 
   // Objects
-  async getObjectsOwnedByAddress(address: string): Promise<SuiObjectInfo[]> {
+  async getObjectsOwnedByAddress(address: SuiAddress): Promise<SuiObjectInfo[]> {
     try {
+      if (!isValidSuiAddress(normalizeSuiAddress(address)) || !address) {
```

Comment:
```suggestion
      if (!address || !isValidSuiAddress(normalizeSuiAddress(address))) {
```
nit: can we put `!address` at the front`?